app/assets/stylesheets/forever_style_guide/modules/_ambassador_banner.scss

```diff
@@ -59,18 +62,13 @@ a.ambassador_banner-no_ambassador {
 
   &.open {
     border-bottom: 2px solid transparentize($color-white, 0.8);
+    & .fa-chevron-down {
+      @include transform(rotate(180deg));
+    }
   }
 }
 
-.ambassador_banner-emphasized {
-  display: block;
-
-  @media screen and (min-width: $screen-xs) {
-    display: inline;
-  }
-}
-
-.ambassador_banner-ambassador_dropdown {
+.ambassador_banner-dropdown {
   position: absolute;
   top: 100%;
```

Comment:
top: 100% is pretty unconventional? v top: 0

Reply:
That gives you a percentage of the height of the containing block. `top:0` would cover the bar entirely, I think we would need to specify a pixel value if we didn't use the percentage? I like it since that's exactly where we want it positioned, directly below the bar. Plus it wouldn't need updated if the height of the bar changed.